apps/omg_watcher/test/fixtures.exs

```diff
@@ -29,7 +29,8 @@ defmodule OMG.Watcher.Fixtures do
   alias OMG.Crypto
   alias OMG.Watcher
   alias OMG.Watcher.DB
-  alias Watcher.TestHelper
+  alias OMG.Watcher.TestHelper
```

Comment:
leftover?

Reply:
unsure, but I'm not really _keen_ on a module named `TestHelper` given mix/elixir also generates a `test_helper.exs` that setups the tests. 🤔